app/src/main/res/layout/fragment_network.xml

```diff
@@ -0,0 +1,89 @@
+    tools:context="com.feedhenry.securenativeandroidtemplate.features.authentication.AuthenticationFragment">
+
+
+    <ImageView
+        android:id="@+id/background"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:adjustViewBounds="false"
+        android:cropToPadding="false"
+        android:scaleType="centerCrop"
+        app:srcCompat="@drawable/fh_background" />
+
+    <TextView
+        android:id="@+id/notes_info"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginLeft="10dp"
+        android:layout_marginRight="10dp"
+        android:layout_marginTop="20dp"
+        android:text=""
+        android:textColor="@color/colorPrimaryDark"
+        android:visibility="visible" />
+
+    <TextView
+        android:id="@+id/permission_info"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_below="@+id/notes_info"
+        android:layout_marginLeft="10dp"
+        android:layout_marginRight="10dp"
+        android:layout_marginTop="20dp"
+        android:textColor="@color/colorPrimaryDark"
+        android:text=""
+        android:visibility="visible" />
+
+    <ProgressBar
+        android:id="@+id/uploadProgress"
+        style="?android:attr/progressBarStyleHorizontal"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_below="@+id/permission_info"
+        android:layout_marginLeft="10dp"
+        android:layout_marginRight="10dp"
+        android:layout_marginTop="20dp"
+        android:visibility="gone" />
+
+    <TextView
+        android:id="@+id/uploadProgressDesc"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_alignLeft="@id/uploadProgress"
+        android:layout_below="@+id/uploadProgress"
+        android:layout_marginRight="10dp"
+        android:layout_marginTop="5dp"
+        android:text=""
+        android:visibility="gone" />
+
+    <Button
+        android:id="@+id/uploadNotesBtn"
+        style="@style/Widget.AppCompat.Button.Colored"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_alignParentBottom="true"
+        android:layout_alignParentStart="true"
+        android:layout_marginBottom="@dimen/authentication_button_margin_bottom"
+        android:enabled="false"
+        android:text="@string/upload_notes" />
+
+    <Button
+        android:id="@+id/cancelUploadBtn"
+        style="@style/Widget.AppCompat.Button.Colored"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_alignParentBottom="true"
+        android:layout_alignParentStart="true"
+        android:layout_marginBottom="@dimen/authentication_button_margin_bottom"
+        android:enabled="false"
```

Comment:
I think this should be `true` by default? The cancel button is only being made visible during the upload process, but the button is not being enabled.

Reply:
yes, it should be `true`